template.yaml

```diff
@@ -72,6 +72,13 @@ Conditions:
       !Equals [build, !Ref Environment],
       !Equals [staging, !Ref Environment],
     ]
+  IsReturnAuthTimeInIdTokenEnabled:
+    !Or [
+      !Equals [dev, !Ref Environment],
+      !Equals [build, !Ref Environment],
+      !Equals [staging, !Ref Environment],
+      !Equals [integration, !Ref Environment],
```

Comment:
Do we want to enable this in integration straight away? Just considering it's an RP facing environment so might want to give them some heads up before we enable it. What do you think?

Reply:
I was under the impression we'd go straight to integration but I'll check